docs/releases/1.19-NOTES.md

```diff
@@ -4,6 +4,8 @@
 
 # Significant changes
 
+* Clusters using the Amazon VPC CNI provider now perform an `ec2.DescribeInstanceTypes` call at instance  launch time. In large clusters or AWS accounts this may lead to API throttling which could delay node readiness. If this becomes a problem please open a GitHub issue.
```

Comment:
```suggestion
* Clusters using the Amazon VPC CNI provider now perform an `ec2.DescribeInstanceTypes` call at instance launch time. In large clusters or AWS accounts this may lead to API throttling which could delay node readiness. If this becomes a problem please open a GitHub issue.
```